### PR TITLE
Issue #19: Adding build bash script; adding pre commit hook

### PR DESCRIPTION
**Pull request by paulo-vieira-centeno**
- Added a bash script to automate steps described in README.md, including setup of githook pathes
- Changed the shell used by githooks/pre-commit to bash shell to avoid errors regarding '[' character
- Updated instructions on README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,26 +26,36 @@ In order to use them, you should rewire the default git hooks folder in your loc
 To generate the database and the go class to handle the connection run the migration tool:
 
 1. Start database and pgadmin using the command `docker-compose up -d postgres pgadmin`
-1. At the scripts/prisma folder run the `generate.sh` file using the command `sh generate.sh`
-1. Your database is started and the tables are generated
+2. At the scripts/prisma folder run the `generate.sh` file using the command `sh generate.sh`
```
Comment (tiagostutz) [inline README.md:29]: The use of "1., 1., 1.,..." for ordered lists is a good practice as if you need to insert any new item you don't have to change all the others with the respective number. Check: https://guides.github.com/features/mastering-markdown/ in te session "Ordered"

So I think we could keep the 
1.
1.
1.

File: README.md [hunk 2]
```diff
@@ -26,26 +26,36 @@ In order to use them, you should rewire the default git hooks folder in your loc
...
+3. Your database is started and the tables are generated
 
-### Build API
+### Build API under Docker
```
Comment (tiagostutz) [inline README.md:32]: What about "Build API using Docker" ?

File: README.md [hunk 3]
```diff
@@ -26,26 +26,36 @@ In order to use them, you should rewire the default git hooks folder in your loc
...
 
 To build stop-analyzing-api run the below code at the project root folder to run with Docker (suggested):
 
 ```
 docker-compose build
 ```
 
-### Run API
+### Run API under Docker
```
Comment (tiagostutz) [inline README.md:40]: The same... change "under" by "using"

**Comment by tiagostutz** [comment]
Hey @zamariola could you clone the @paulo-vieira-centeno branch, run locally and merge this PR when it is good to go?

**Comment by tiagostutz** [comment]
@paulo-vieira-centeno I just invited you to join the team here. 
I hope you could accept it! =)

**Comment by zamariola** [comment]
Sure! I can do it today or tomorrow, for sure.

**Comment by zamariola** [comment]
It seems ok to me! It's working fine locally.

File: README.md (hunk 1 above)
Comment (paulo-vieira-centeno) [reply README.md:29]: Yes, Ok

**Comment by paulo-vieira-centeno** [comment]
@tiagostutz Thanks for the invitation, I've accepted it, of course.
I'll take a look at the things you and the rest pointed in my P.R. as soon as possible, it's been a busy week... :)

**Pull request merged**